src/main/java/openmodularturrets/tileentity/turretbase/TurretBase.java

```diff
@@ -153,7 +150,7 @@ public double injectEnergy(ForgeDirection forgeDirection, double v, double v1) {
     @Optional.Method(modid = "IC2")
     @Override
     public int getSinkTier() {
-        return 4;
+        return Integer.MAX_VALUE;
```

Comment:
Can you explain the reasoning behind this change?

Reply:
Might be a misinterpretation by me. I want the turret base to accept every voltage.

Reply:
Okay, I got it. We'll reverse this change if it causes any problems.